CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift

```diff
@@ -381,6 +404,61 @@ public final class Camera2ViewController: UIViewController, CameraScreen {
         return finalImage
     }
     // swiftlint:enable line_length
+
+    private func showQRCodeFeedback(for document: GiniQRCodeDocument) {
+        hideTask = DispatchWorkItem(block: {
```

Comment:
[Suggestion] Could you please refactor this method and extract two methods for the successful and unknown QRCodes?